aws_lambda_scripts/main.py

```diff
@@ -22,81 +23,110 @@
 object_name = "historic_usage_data.json"
 file_name = "/tmp/historic_usage_data.json"
 
+logger = logging.getLogger()
 
 def handler(event, context):
     file_exists = True
 
+    logger.info("Starting process")
+
+    logger.info("Creating S3 client")
+
     # Create an S3 client
     session = boto3.Session()
     s3 = session.client('s3')
 
-    print("Session created")
+    logger.info("S3 client created")
+
+    logger.info("Getting historic_usage_data.json from S3")
 
     # Get historic_usage_data.json from S3
     try:
         s3.download_file(bucket_name, object_name, file_name)
     except ClientError as e:
-        print(f"Error getting historic_usage_data.json from S3: {e}")
+        logger.exception("Error getting historic_usage_data.json from S3")
         file_exists = False
     else:
-        print("Downloaded historic_usage_data.json from S3")
+        logger.info("Downloaded historic_usage_data.json from S3")
```

Comment:
Better to use object_name variable than hardcode the filename. Multiple uses in this file.